tests/test_api.py

```diff
@@ -30,9 +30,9 @@ def test_loads_region_bestvar(client):
 
     assert response.status_code == 200
     content = response.get_json()
-    assert content["data"]["tissue"] == "thyroid"
-    assert content["data"]["study"] == "GTEx"
-    assert content["data"]["symbol"] == "GSTM1"
+    assert content["data"]["tissue"] == "adipose_naive"
+    assert content["data"]["study"] == "FUSION"
+    assert content["data"]["symbol"] == "AMIGO1"
```

Comment:
Should I update the data pipeline "auto subset" code to reflect a new set of regions? Do you have a list of regions that you were (manually) using when you prepared this test data?

Reply:
One of the homepage suggested regions is not available in test data. Is this intentional (eg we decided there were better examples)?

`chr2:21042269 `

Reply:
The one region on chromosome 1 that's being tested (1:108774968-109774968) is the same -- the previous results were simply incorrect. Our paper no longer references the results from chromosome 2, so maybe we should remove all references to chr2 altogether, along with test data for it.

Reply:
Certainly, it makes sense to update the homepage if we no longer think the examples are relevant/interesting.